inception-sharing/src/main/java/de/tudarmstadt/ukp/inception/sharing/ProjectInvite.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt 
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *  
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.sharing;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "project_invite")
+public class ProjectInvite
+{
+    @Id
+    @Column(nullable = false)
```

Comment:
I am surprised it works without adding `@GeneratedValue(strategy = GenerationType.IDENTITY)` as e.g. in `SourceDocument`.

Reply:
It's not a generated value, it's the (unique) project ID.

Reply:
The primary key of your table is simultaneously a foreign key? Uh - I guess that might work, but it feels very very odd. I think the project should be a separate column (i.e a field of type Project) with a corresponding foreign key to the project table. That's how we usually do it.